### PR TITLE
fix:  Add domain name validation when configuring ingress to loadbalancer

### PR DESCRIPTION
**Pull request by mikecirioli**
#### Submitter checklist

- [x] Change is code complete and matches issue description.
- [x] Change is covered by existing or new tests.

#### Description
Adds simple domain name validation (domain names must not contain any whitespace) when configuring LB ingress during installation.  Also adds unit test for `NoWhiteSpaceValidator`

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: pkg/jx/cmd/init.go [hunk 1]
```diff
@@ -925,3 +925,19 @@ func (o *CommonOptions) GetDomain(client kubernetes.Interface, domain string, pr
 
 	return domain, nil
 }
+
+//NoWhiteSpaceValidator is an input validator for the survey package that disallows any whitespace in the input
+func NoWhiteSpaceValidator() survey.Validator {
```
Comment (ccojocar) [inline pkg/jx/cmd/init.go]: I would be nice if you could move this function in the `util/survey.go`, that can be reused in other parts of the code.

Comment (mikecirioli) [reply pkg/jx/cmd/init.go]: that was actually something i was going to ask about this morning :)

**Pull request merged**